### PR TITLE
Disable some failing/hanging HTTP tests

### PR DESCRIPTION
**Pull request by davidsh**
Opened some issues #28423 #28424. These tests are failing/hanging on my dev box all the time.

### CODE REVIEW

**Comment by davidsh** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build

File: src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ResponseDrain.cs [hunk 1]
```diff
@@ -198,6 +198,7 @@ protected static string GetResponseForContentMode(string content, ContentMode mo
 
         // Similar to above, these are semi-extreme cases where the response should never drain for any handler.
 
+        [ActiveIssue(28423)]
```
Comment (stephentoub) [inline src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ResponseDrain.cs]: It's failing/hanging with all handlers?

Comment (davidsh) [reply src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ResponseDrain.cs]: I thought it was both WinHttpHandler and SocketsHttpHandler.  But further testing shows just WinHttpHandler having issues.  

I will revise the disabling of the test.

**Comment by davidsh** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build

**Comment by davidsh** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Linux x64 Debug Build

**Comment by davidsh** [comment]
CI failures unrelated.

**Pull request merged**